docs/server/source/simple-network-setup.md

```diff
@@ -194,14 +217,37 @@ persistent_peers = "<Member 1 node id>@<Member 1 hostname>:46656,\
 <Member N node id>@<Member N hostname>:46656,"
 ```
 
-Now the Member must run two commands:
-- `bigchaindb start`
-- `tendermint node`
+## Member: Start MongoDB, BigchainDB and Tendermint
+
+To start MongoDB, one uses the command `mongod` but that will run it in the foreground. If you want to run it in the background (so it will continue running after you logout), you can use `mongod --fork --logpath /var/log/mongodb.log`. (You might have to create the `/var/log` directory if it doesn't already exist.)
+
+The _recommended_ approach is to reboot the machine and let the MongoDB startup script start MongoDB. Don't do that yet though…
```

Comment:
Since when does `apt install anything` _start_ the thing in question? Is that common? I had no idea that that sometimes happens. It seems like overloading the word "install" to mean "install and start too".

I know that installing MongoDB from the official MongoDB PPA does _not_ also start MongoDB. Moreover, we've always recommended installing MongoDB from the official PPA in the past, because the MongoDB you get by using `apt install mongodb` can get behind the latest official MongoDB. Now I'm feeling even more wary of the package that the Ubuntu people built. What _else_ does their so-called "installer" do?

Reply:
Anyway, I guess I'll add something about how the so-called installer also starts MongoDB, and I'll remove what I wrote about how to start it. In the future, once there's an official MongoDB package for Ubuntu 18.04, I think we should go back to using that. Maybe I'll also file a bug report with Ubuntu: "BUG: Installer doesn't just install, it also starts MongoDB. Do not want.".